app/jobs/resolution_proofing_job.rb

```diff
@@ -158,64 +105,9 @@ def logger_info_hash(hash)
     logger.info(hash.to_json)
   end
 
-  def user_can_pass_after_state_id_check?(resolution_result)
-    return true if resolution_result.success?
-    # For failed IV results, this method validates that the user is eligible to pass if the
-    # failed attributes are covered by the same attributes in a successful AAMVA response
-    # aka the Get-to-Yes w/ AAMVA feature.
-    return false unless resolution_result.failed_result_can_pass_with_additional_verification?
-
-    attributes_aamva_can_pass = [:address, :dob, :state_id_number]
-    results_that_cannot_pass_aamva =
-      resolution_result.attributes_requiring_additional_verification - attributes_aamva_can_pass
-
-    results_that_cannot_pass_aamva.blank?
-  end
-
   def resolution_proofer
-    @resolution_proofer ||=
-      if IdentityConfig.store.proofer_mock_fallback
-        Proofing::Mock::ResolutionMockClient.new
-      else
-        Proofing::LexisNexis::InstantVerify::Proofer.new(
-          instant_verify_workflow: IdentityConfig.store.lexisnexis_instant_verify_workflow,
-          account_id: IdentityConfig.store.lexisnexis_account_id,
-          base_url: IdentityConfig.store.lexisnexis_base_url,
-          username: IdentityConfig.store.lexisnexis_username,
-          password: IdentityConfig.store.lexisnexis_password,
-          request_mode: IdentityConfig.store.lexisnexis_request_mode,
-        )
-      end
-  end
-
-  def lexisnexis_ddp_proofer
-    @lexisnexis_ddp_proofer ||=
-      if IdentityConfig.store.lexisnexis_threatmetrix_mock_enabled
-        Proofing::Mock::DdpMockClient.new
-      else
-        Proofing::LexisNexis::Ddp::Proofer.new(
-          api_key: IdentityConfig.store.lexisnexis_threatmetrix_api_key,
-          org_id: IdentityConfig.store.lexisnexis_threatmetrix_org_id,
-          base_url: IdentityConfig.store.lexisnexis_threatmetrix_base_url,
-        )
-      end
-  end
-
-  def state_id_proofer
-    @state_id_proofer ||=
-      if IdentityConfig.store.proofer_mock_fallback
-        Proofing::Mock::StateIdMockClient.new
-      else
-        Proofing::Aamva::Proofer.new(
-          auth_request_timeout: IdentityConfig.store.aamva_auth_request_timeout,
-          auth_url: IdentityConfig.store.aamva_auth_url,
-          cert_enabled: IdentityConfig.store.aamva_cert_enabled,
-          private_key: IdentityConfig.store.aamva_private_key,
-          public_key: IdentityConfig.store.aamva_public_key,
-          verification_request_timeout: IdentityConfig.store.aamva_verification_request_timeout,
-          verification_url: IdentityConfig.store.aamva_verification_url,
-        )
-      end
+    @resolution_proofer ||= Proofing::Resolution::ProgressiveProofer.new
+    @resolution_proofer
```

Comment:
second line is redundant
```suggestion
    @resolution_proofer ||= Proofing::Resolution::ProgressiveProofer.new
```